src/validator_main.cpp

```diff
@@ -134,9 +134,16 @@ namespace
                 std::string errortype = (out == ebi::vcf::SUMMARY) ? "errors_summary" : "errors";
                 std::string filename = input + "." + errortype + "." + std::to_string(timestamp) + "." + filetype;
                 boost::filesystem::path file{filename};
-                if (boost::filesystem::exists(file)) {
-                    throw std::runtime_error{"Report file already exists on " + filename + ", please delete it or rename it"};
+
+                try {
+                    if (boost::filesystem::exists(file)) {
+                        throw std::runtime_error{"Report file already exists on " + filename + ", please delete it or rename it"};
+                    }
+                } catch (boost::filesystem::filesystem_error const & er) {
+                    // commented it to let it be caught in RepportWriter
+                    // throw std::runtime_error{"Permissions denied, can't access the location " + filename};
```

Comment:
after comparing how it behaves in develop and here, the develop behaviour is better, as the boost exception explains better the problem than we. please revert every change in this file.